opencog/cogserver/scm/config.scm

```diff
@@ -44,6 +44,7 @@
 (add-to-ltdl-path (string-append (getcwd) "/opencog/nlp/sureal"))
 (add-to-ltdl-path (string-append (getcwd) "/opencog/nlp/lg-dict"))
 (add-to-ltdl-path (string-append (getcwd) "/opencog/planning"))
+(add-to-ltdl-path (string-append (getcwd) "/opencog/scm/opencog/learning"))
```

Comment:
FYI, https://stackoverflow.com/questions/33061198/what-is-the-difference-betweend-ltdl-library-path-and-ld-library-path